app/autoload.php

```diff
@@ -2,6 +2,8 @@
 
 use Doctrine\Common\Annotations\AnnotationRegistry;
 
+error_reporting(error_reporting() & ~E_USER_DEPRECATED);
+
 if (!$loader = @include __DIR__.'/../vendor/autoload.php') {
```

Comment:
I suggest either changing this to ``require``, as done in Symfony, or at least ensuring that you get a failure exit code (i.e. displayign the message and then using ``exit(1)``) if you want to keep the nicer error message